### PR TITLE
Resolves issue#718

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- Let the repository require `php-7.0` version at least because the PHP 7 has been released for whiles.
- Change into the `7.0.0` in `PHPUnit` attribute setting.
- Using the `expectException` since the `PHPUnit` version is `6.5+`.

### CODE REVIEW

**Comment by lukebakken** [comment]
@ramunasd considering `5.6` reached end-of-life [this year](https://www.php.net/supported-versions.php) is it reasonable to drop support already?

**Comment by peter279k** [comment]
> @ramunasd considering `5.6` reached end-of-life [this year](https://www.php.net/supported-versions.php) is it reasonable to drop support already?

I think it's ok because the `php-5.6` has been end of life this year.

I think developers should migrate them to `php-7.0` at this moment.

**Comment by lukebakken** [comment]
@peter279k the problem are users who start with operating system images that do not have recent PHP releases available. For instance, Ubuntu [14.04 LTS end-of-life](https://wiki.ubuntu.com/Releases) is in 2022 and PHP `5.5.9` is the only one available out-of-the-box (output truncated):

```
root@ubuntu-14:~# apt-get install php
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package php

root@ubuntu-14:~# apt-get install php7
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package php7

root@ubuntu-14:~# apt-get install php5
Reading package lists... Done
Building dependency tree
Reading state information... Done

root@ubuntu-14:~# php --version
PHP 5.5.9-1ubuntu4.29 (cli) (built: Apr 22 2019 18:33:52)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend OPcache v7.0.3, Copyright (c) 1999-2014, by Zend Technologies
```

**Comment by peter279k** [comment]
Hi @lukebakken, thanks for your reply.

If we want to let the package be out-of-the-box.

I think we can fallback to support the `php-5.6` version.

**Comment by ramunasd** [comment]
@lukebakken I would like to drop support for old PHP versions gradually. We already lifted up minimum version for upcomming release.
Also, would be nice to have compatibility matrix. @peter279k could You please?

**Comment by peter279k** [comment]
Hi @ramunasd, thanks for your reply.

Should we fallback to the `php-5.6` version requirement?

Is there anything work I need to be considered?

**Comment by ramunasd** [comment]
please leave 5.6 support

**Comment by peter279k** [comment]
@ramunasd, ok. Thanks for your reply.

What's the minimum PHP version support? `php-7.0`?

If we set this `php-7.0` version support is the minimum requirement, I think the PR has been already done.

File: tests/Functional/Channel/DirectExchangeTest.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ public function basic_consume_foo()
             $this->assertEquals($this->message->properties['content_type'], $msg->get('content_type'));
             $this->assertEquals($this->message->properties['correlation_id'], $msg->get('correlation_id'));
             $this->assertEquals($this->message->properties['reply_to'], $msg->get('reply_to'));
-            $this->setExpectedException('OutOfBoundsException');
+            $this->expectException('OutOfBoundsException');
```
[the remark this answers is not in the document]
Comment (peter279k) [reply tests/Functional/Channel/DirectExchangeTest.php]: @ramunasd, this has the different approaches on `PHPUnit 5` and `PHPUnit 6+`.

The `setExpectedException` method is available on `PHPUnit 5`.

And the `PHPUnit 6+` change into `expectException` method.

The `PHPUnit 5` supports the `php-5.6`.

To be compatible with `php-5.6`, I think we can use the `expectedException` comment annotation.

And The unit tests will be passed/done for supported PHP versions.

**Pull request merged**